### ISSUE TITLE
Using contigs instead of scaffolds for the genome extraction workflow

### ISSUE THREAD

**Issue by adityabandla** [MadsAlbertsen/mmgenome#27]
Hi,

I used MEGAHIT to assemble a sediment metagenome. However, I ran into issues with finding a suitable scaffolder for converting the contigs to scaffolds. 

Can I go ahead with the mmgenome workflow only with contigs? Is there a recommended scaffolder for metagenomic contigs?

Thanks

**Comment by Kirk3gaard** [comment]
Hi Aditya Bandla

There is no need for scaffolding. The package works equally well for contigs.

**Comment by adityabandla** [comment]
Thanks Rasmus! I sequenced 27x samples across 2 lanes on the HiSeq (250bp). These samples are spatially related and I see differences in the relative proportions of my target organism  (whose genomes I am trying to assemble).

I did a composite assembly i.e. combined reads across all 27x samples into a single file which was then assembled. I then mapped each sample individually back to the contigs to estimate coverage. Is this considered a valid approach?

Thanks for the advise!
